### DISCUSSION TITLE
Sveltekit with vite 4.0

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#530] [General]
Hi there is a conflict dev dependency error when using sveltekit with vite 4

```
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.2.0" from @sveltejs/kit@1.0.0-next.577
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"1.0.0-next.577" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: vite@3.2.5
npm ERR! node_modules/vite
npm ERR!   peer vite@"^3.2.0" from @sveltejs/kit@1.0.0-next.577
npm ERR!   node_modules/@sveltejs/kit
npm ERR!     dev @sveltejs/kit@"1.0.0-next.577" from the root project
npm ERR!
```

**Comment by ivanhofer** [comment]
This has nothing to do with `typesafe-i18n`. Or do I miss something?

**Reply by bitdom8** [reply]
If you could just try to download and install the latest vite with npm-check-update, this happens. Thanks for great repo and quick response @ivanhofer

**Reply by ivanhofer** [reply]
Can confirm. This is nothing `typesafe-i18n`-related. You just need to install `vite` 4 first and then upgrade `SvelteKit`. If you install it simultaneously, you'll get an error.

**Reply by bitdom8** [reply]
Thanks @ivanhofer, that will nicely do it then. Spasiba
